src/lru.js

```diff
@@ -80,7 +104,7 @@ class LRU {
 				this.delete(key);
 			} else {
 				result = item.value;
-				this.set(key, result, true);
```

Comment:
bumpLru always behaves as if bypass is true, so parameter is not needed. And since key is only used to resolve an item, we can just pass the item itself